akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/SyncTestingExampleSpec.scala

```diff
@@ -93,8 +93,9 @@ class SyncTestingExampleSpec extends WordSpec with Matchers {
       //#test-child-message-anonymous
       val testKit = BehaviorTestKit(myBehavior)
       testKit.run(SayHelloToAnonymousChild)
-      // Anonymous actors are created as: $a $b etc
-      val childInbox = testKit.childInbox[String](s"$$a")
+      val child = testKit.expectEffectType[SpawnedAnonymous[String]]
```

Comment:
could we carry the inbox in the `SpawnedAnonymous`?

Reply:
Would it make it harder to assert against but it already contains a Behavior which is hard for anything other than static behaviors.

Reply:
Not easily since `SpawnedAnonymous` is in `akka.actor.testkit.typed` while `TestInbox` is separate between scaladsl/javadsl.